corehq/apps/app_manager/views/utils.py

```diff
@@ -411,3 +413,18 @@ def clear_xmlns_app_id_cache(domain):
 
 def form_has_submissions(domain, app_id, xmlns):
     return FormES().domain(domain).app([app_id]).xmlns([xmlns]).count() != 0
+
+
+def get_multimedia_sizes_for_build(domain, build_id, build_profile_id=None):
+    build = get_app_cached(domain, build_id)
+    assert build.copy_of, _("Size calculation available only for builds")
```

Comment:
Is this needed? What happens if you put a non-build id into the URL for view source files?

Reply:
Will the page even work? Isn't this view only for builds?

Reply:
Tried it out - View Source Files does work if you give it a primary (non-build) app id, even if the app has unpublished changes. I haven't looked closely enough to see if the build files include the unpublished changes (I assume they don't). Anyway, I think you could leave these assertions off, I think it's fine to compare a build with the current app (since making a build doesn't change multimedia), but I don't feel that strongly either way.

Reply:
The reason for keeping this to builds was that the mm for the current app is not frozen i.e it still can change. So finding MM difference for an app would not make sense.
Plus, you won't get app_id on that page unless someone directly changes the URL, should we worry about such actions?

Reply:
Up to you, I don't think it's a big deal.